test/ERC6538Registry.t.sol

```diff
@@ -305,6 +332,24 @@ contract RegisterKeysOnBehalf is ERC6538RegistryTest {
     registry.registerKeysOnBehalf(registrant, schemeId, signature, stealthMetaAddress);
   }
 
+  function testFuzz_RevertIf_AManipulatedErc712SignatureIsUsedToRegisterADifferentStealthMetaAddress(
+    string memory registrantSeed,
+    uint256 schemeId,
+    bytes memory stealthMetaAddress,
+    bytes memory attackerStealthMetaAddress
+  ) external {
+    vm.assume(keccak256(stealthMetaAddress) != keccak256(attackerStealthMetaAddress));
+    (address registrant, uint256 registrantPrivateKey) = makeAddrAndKey(registrantSeed);
+    bytes memory signature =
+      _generateRegistrationSignature(registrantPrivateKey, schemeId, stealthMetaAddress, 0);
+    bytes memory manipulatedSignature = manipulateSignature(signature);
+
+    vm.expectRevert(bytes(""));
+    registry.registerKeysOnBehalf(
+      registrant, schemeId, manipulatedSignature, attackerStealthMetaAddress
+    );
```

Comment:
Why/where exactly does this revert? I thought the manipulated signature with the different s value should be valid as long as the original was never used

Reply:
It's reverting because the "attacker" is trying to register a different `stealth meta address` with a manipulated signature